test/controllers/HomepageControllerSpec.scala

```diff
@@ -57,7 +60,10 @@ class HomepageControllerSpec extends FrontEndTestHelper {
     }
 
     "render the homepage page with an authenticated standard user" in {
-      val controller = new HomepageController(getAuthorisedSecurityComponents, getValidStandardUserKeycloakConfiguration, consignmentService)
+      val config: Config = ConfigFactory
```

Comment:
Should make the the `false` the default config, as this will be how the page will be once the feature is released. 

The tests testing the page without the button should be the ones which override the config